tests/Filter/StringFilterTest.php

```diff
@@ -80,6 +83,9 @@ public function testContains($containsType): void
         $this->assertTrue($filter->isActive());
     }
 
+    /**
+     * @return array<array{int|null}>
```

Comment:
```suggestion
     * @return array<array{int}, array{null}>
```

Reply:
I wasn't happy with this one and I've changed it a little bit to add more cases and be more explicit.